meshroom/core/attribute.py

```diff
@@ -283,6 +283,8 @@ def defaultValue(self):
         return copy.copy(self.desc.value)
 
     def _isDefault(self):
+        if self.isLink:
+            return self.getLinkParam().value == self.defaultValue()
         return self._value == self.defaultValue()
```

Comment:
```suggestion
        return self.value == self.defaultValue()
```
Not fully sure if it's evaluated properly, else:
`self._get_value() == self.defaultValue()`

Reply:
Done